### ISSUE TITLE
Added a Signal Strength Function

### ISSUE THREAD

**Issue by graemerae** [adafruit/Adafruit_CircuitPython_RockBlock#9]
Thought you might want to fold this in at some point.  Returns a Signal Quality from 0-5

```
    def signalStrength(self):
        resp = self._uart_xfer("+CSQ")
        if resp[-1].strip().decode() == "OK":
            return resp[1].strip().decode().split(":")[1]
        return None
```

**Comment by dherrada** [comment]
@graemerae It'd be great if you could make a PR to add this if you're up to it. Thanks!

**Comment by graemerae** [comment]
I suck at PR’s  - I tried a couple of times, but the indentation is all screwed up.

**Comment by caternuson** [comment]
Closing. This was added with #15 and released with 1.2.0. Look for the new `signal_quality` property.
